pkg/admission/namespaceconditions/decorator.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // this is a list of namespaces with special meaning.  The kube ones are here in particular because
 // we don't control their creation or labeling on their creation
-var runLevelZeroNamespaces = sets.NewString("default", "kube-system", "kube-public")
+var runLevelZeroNamespaces = sets.NewString("kube-system", "kube-public")
```

Comment:
> liggitt 4 days ago  Member
> looking at SkipRunLevelZeroPlugins and SkipRunLevelOnePlugins plugins, I wondered about openshift.io/RestrictSubjectBindings in the default namespace... from what I can see, we got lucky and the registry/router setups create ClusterRoleBindings, not RoleBindings, but wanted a second set of eyes on those (since the setups only run once, and won't retry in a loop that would succeed after the openshift API was available)

Choose your second set of eyes?  I don't imagine they'd be changing their permissions and they do, they'll notice and tell us.

Reply:
on review, the registry and router setup do not create namespaced roles/rolebindings, so the RestrictSubjectBindings admission plugin wouldn't block those